pyaedt/application/Design.py

```diff
@@ -3739,6 +3739,8 @@ def save_project(self, file_name=None, overwrite=True, refresh_ids=False):
         refresh_ids : bool, optional
             Whether to refresh object IDs after saving the project.
             The default is ``False``.
+        create_parent_dir : bool, optional
+            Whether to create the parent directory of file_name if it does not exist.
```

Comment:
```suggestion
            Whether to create the parent directory for the project if it does not exist.
```
Whether than use the code entity, consider rewriting in plain English: Whether to create the parent directory of the ``file_name`` parameter
            if it does not exist.